sites/main-page-title/_base.scss

```
@@ -32,4 +32,9 @@ hgroup#wb-cont {
 	h1 {
 		margin-top: 0;
 	}
+
+	p:last-child {
```

See the inner comment in the main-page-title include file

```suggestion
	&with-byline p:last-child {
```


```suggestion
	p:last-child.gc-byline {
```